node_modules/oae-authentication/tests/test-auth-local.js

```diff
@@ -1173,55 +1205,75 @@ describe('Authentication', function() {
+                                    assert.ok(err);
+                                    assert.equal(err.code, 400);
+                                    assert.ok(!user);
 
-                                // Verify the context was not authenticated
-                                RestAPI.User.getMe(testGlobalUserRestContext, function(err, me) {
-                                    assert.ok(!err);
-                                    assert.ok(me.anon);
+                                    // Ensure the global admin still cannot be authenticated
+                                    var testGlobalUserRestContext = TestsUtil.createGlobalRestContext();
 
-                                    // Sanity check the user can be created through the REST endpoints
-                                    RestAPI.User.createGlobalAdminUser(globalAdminRestContext, username, username, username, null, function(err, user) {
-                                        assert.ok(!err);
-                                        assert.ok(user);
+                                    // Ensure that the credentials do not authenticate a value global administrator
+                                    RestAPI.Authentication.login(testGlobalUserRestContext, username, username, function(err) {
+                                        assert.ok(err);
+                                        assert.equal(err.code, 401);
 
-                                        // Ensure when we try and create one with the same loginId, we get a 400 error
-                                        RestAPI.User.createGlobalAdminUser(globalAdminRestContext, username, username, username, null, function(err, secondUser) {
-                                            assert.ok(err);
-                                            assert.equal(err.code, 400);
+                                        // Verify the context was not authenticated
+                                        RestAPI.User.getMe(testGlobalUserRestContext, function(err, me) {
+                                            assert.ok(!err);
+                                            assert.ok(me.anon);
 
-                                            return callback();
+                                            // Sanity check the user can be created through the REST endpoints
+                                            RestAPI.User.createGlobalAdminUser(globalAdminRestContext, username, username, username, email, {}, function(err, user) {
+                                                assert.ok(!err);
+                                                assert.ok(user);
+
+                                                // Ensure when we try and create one with the same loginId, we get a 400 error
+                                                RestAPI.User.createGlobalAdminUser(globalAdminRestContext, username, username, username, email, {}, function(err, secondUser) {
```

Comment:
I guess we could say that you can only invite new users by email address and have to select the person you want from the autosuggest if they already exist. Would that be a viable approach?